### ISSUE TITLE
failed switching to "991:991": operation not permitted

### ISSUE THREAD

**Issue by zevlee** [spantaleev/matrix-docker-ansible-deploy#953]
I'm getting an error preventing ``matrix-synapse.service`` from starting, as shown below. Manually setting ``matrix_user_uid`` and ``matrix_user_gid`` did not work for me.

```
Mar 19 09:25:57 server systemd[1]: Starting Synapse server...
Mar 19 09:25:57 server systemd[1]: Started Synapse server.
Mar 19 09:26:00 server matrix-synapse[234025]: error: failed switching to "991:991": operation not permitted
```

**Comment by 0x46616c6b** [comment]
Same here.

**Comment by jokey2k** [comment]
Encountered same issue, workaround is fine. though stop all services at first, then work through the list, otherwise the Postgres might crash as it cannot write its datafilee properly in the transition

**Comment by aaronraimist** [comment]
Should be fixed by https://github.com/spantaleev/matrix-docker-ansible-deploy/commit/f99dcd611fadce7d20a3a804b4aac49d2d3781e2

**Comment by zevlee** [comment]
> Should be fixed by [f99dcd6](https://github.com/spantaleev/matrix-docker-ansible-deploy/commit/f99dcd611fadce7d20a3a804b4aac49d2d3781e2)

I can confirm that the issue was fixed.
